src/data/store.js

```diff
@@ -45,8 +42,15 @@ const segmentMiddleware = createMiddleware(eventsMap, Segment());
 
 const middleware = [thunkMiddleware, loggerMiddleware, routerHistoryMiddleware, segmentMiddleware];
 
+const initialState = apiClient.getAuthenticationState();
+if (initialState.authentication) {
+  // eslint-disable-next-line no-undef
+  analytics.identify(initialState.authentication.email);
```

Comment:
Rather than calling `analytics` directly can we change this to something like `identifyUser(email => { userId: email})(initialState.authentication.email);`

Since the rest of the tracking is going through beacon it seems to make sense to have this also use beacons identifyUser event, even if we just call it immediately.